MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -64,6 +65,30 @@ def store_issue_details(issue, repo, db):
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	content_extracted = extract_jenkins_link_and_testname(issue['body'])
+	issue_quoted_jenkins_links = content_extracted[0]
+	issue_quoted_test_names = content_extracted[1]
+	if issue_quoted_jenkins_links != []:
+    		issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+	if issue_quoted_test_names != []:
+    		issue_details['issue_quoted_test_names'] = issue_quoted_test_names
+
+	if (jenkins_links and testnames):
+		for link in jenkins_links:
+		    for test in test_names:
+			new_string = link.split('/')
+			final_string = ''.join(new_string)
+			issue_uid2 = f'{repo_name}_{issue_number}_{final_string}_{test}.txt'
+			jenkins_output_path = f'./data/GitHubData/JenkinsOutput/{issue_uid2}'
+
+			if not os.path.isfile(issue_uid2):
+			    trss_jenkins_output = query_trss_for_jenkins_output(link, test)
+			    if trss_jenkins_output:
+				store_on_fs(trss_jenkins_output, jenkins_output_path)
+				issue_details['jenkins_output_path'] = [trss_jenkins_output]
```

Comment:
For here, you saved `output` into `path`, we need to save the `path of the output` to `path` instead.
Also, we need a list to store all paths from the same issue, instead of putting each path into a list. Thanks.